mobile/library/cc/engine_builder.h

```diff
@@ -215,7 +216,7 @@ class EngineBuilder {
 
   LogLevel log_level_ = LogLevel::info;
   absl::optional<envoy_logger> envoy_logger_;
-  EngineCallbacksSharedPtr callbacks_;
+  std::unique_ptr<InternalEngineCallbacks> callbacks_;
```

Comment:
nice that we can make this a unique_ptr!  was it shared_ptr before because it was used in C too?

Reply:
Yup, it was a `shared_ptr` before.